src/CMakeLists.txt

```diff
@@ -0,0 +1,659 @@
+cmake_minimum_required(VERSION 3.2)
+
+set(PRJ_NAME "tvm")
+project(${PRJ_NAME} C CXX)
```

Comment:
Is there a reason for using nested projects? Seems like it adds unnecessary complication.